src/utils/onlineParser.js

```diff
@@ -280,7 +265,7 @@ function unsuccessful(state) {
 }
 
 // Given a stream, returns a { kind, value } pair, or null.
-function lex(LexRules, stream) {
+function getMatch(LexRules, stream) {
```

Comment:
I think `lex` is a better name of this. You could call it `getLexedToken` if you want a longer name.